hw20/roles/hw201/tasks/docker.yml

```diff
@@ -0,0 +1,55 @@
+---
+#Ubuntu
+- name: "Install Docker on Ubuntu"
+  block:
+    - name: "Set up the repository"
+      ansible.builtin.package:
+       name: "{{ item }}"
+       update_cache: yes
+      loop:

```

Comment:
Use variable here. This task can be applied for both Ubuntu and Centos

Reply:
fix